app/hooks/projects/types.ts

```diff
@@ -1,5 +1,15 @@
 import { AxiosRequestConfig } from 'axios';
+import { Response } from 'types/api-model';
 
+// useProjects
+export interface UseProjectsProps {
+  search?: string;
+}
+export interface UseProjectsResponse extends Response {
```

Comment:
instead of using your own type Response I think you should use `UseQueryResult` from react-query

https://github.com/tannerlinsley/react-query/blob/master/src/react/types.ts#L42

Reply:
That change wouldn't be trivial. We would need to do something like this: https://blog.johnnyreilly.com/2021/01/strongly-typing-react-query-s-usequeries.html

_It wouldn't as simple as creating an interface that extends `UseQueryResult` since that can't be done just like that_